components/Feature/VulnerabilitiesGrid/ResourceCard/index.js

```diff
@@ -33,7 +32,8 @@ const ResourceCard = ({
   const distributionElement =  tags.filter(t => HIDDEN_TAGS.includes(t)).join(", ")
   const tagsElement = tags.filter(t => !HIDDEN_TAGS.includes(t)).map(item=> (<span key={"tags-"+item} className={css.tags}>{trimLength(item, 20)}</span>))
   const snapshot = (customerId != undefined) ? true : false
-  const foodResource = (foodNeedsId == 'food-needs') ? true : false
+  const summaryDataExists = ( telephone  || distance != 100 || currentProvision || openingTimes) ? true : false
```

Comment:
I think you can do this without the `? true : false` on the end, as the first part of it already gives you a true/false?

```
> 1 + 1 = 2
true
> 1 + 1 = 2 ? true : false
true
```